v3/tags_data.go

```diff
@@ -564,9 +564,18 @@ IFD:
   name: FNumber
 # NOTE(dustin): SRATIONAL isn't mentioned in the standard, but we have seen it in real data.
   type_names: [RATIONAL, SRATIONAL]
+- id: 0x830e
```

Comment:
Please move all of these to the bottom and put a comment about what they are, and a URL to the source.